src/redux/configureStore.js

```diff
@@ -0,0 +1,10 @@
+import { combineReducers } from 'redux';
+import booksReducer from './books/books';
+import catReducer from './categories/categories';
+
+combineReducers = ({
+  book: booksReducer,
+  category: catReducer,
+});
+
+export default combineReducers;
```

Comment:
- [ ] Great Job using store, actions, and reducers 💯 But according to the requirements you should combine both reducers into a root reducer by using configureStore function, so Kindly fix it 👌

> For example 💡
```
const rootReducer = combineReducers = ({
  book: booksReducer,
  category: catReducer,
});

const store = configureStore({
  reducer: rootReducer,
});

export default store;
```

![image](https://user-images.githubusercontent.com/45231276/184862052-69bb98df-9049-41cc-a50c-e6238dbfb157.png)